.github/workflows/ci.yml

```diff
@@ -96,6 +96,17 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: -- -A clippy::uninlined-format-args
 
+  security:
+    if: github.ref != 'refs/heads/staging'
+    needs: sanity
+    runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
If I understand correctly, the continue-on-error would mean a dependancy audit warning would not block the CI, right? It would be treated as a success for `bors merge`?

Reply:
From what I read, yes.